src/components/ChatRoom/MessengerContainer.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useRef, useEffect } from 'react';
+import {
+  Container,
+  MessengerHeaderContainer,
+  MessengerContentContainer,
+  UserProfile,
+  MessengerTextFormContainer,
+  Form,
+  TextInput,
+  ChatContainer,
+  Chat,
+  ChatBubble,
+  TextBox,
+  TimeBox,
+  HeartContainer,
+} from './MessengerPresenter';
+
+import { ProfileImage, Button, HeartImg } from '../icons';
+import dateToString from '../../utils/date';
+
+const MessengerContainer = ({ messengerData, onSubmit }) => {
+  const [text, setText] = useState('');
+  const [isMe, setIsMe] = useState(true);
+
+  //scroll 조절
+  const scrollRef = useRef();
+  const [chatEdit, setChatEdit] = useState(false);
+
+  function scrollToBottom() {
+    scrollRef.current.scrollTo({
+      top: scrollRef.current.scrollHeight,
+      behavior: 'smooth',
+    });
+  }
+  useEffect(() => {
+    scrollToBottom();
+  }, [chatEdit]);
+
+  const handleSubmitButtonClick = () => {
+    // form이 clear되면서 todoList에 값이 추가되어야 함.
+    if (!text) {
+      window.alert('공백 메시지는 보낼 수 없어요~');
+    } else {
+      onSubmit({
+        id: isMe,
+        text: text,
+        date: new Date().getTime(),
+        isHeart: false,
+      });
+      setText('');
+      setChatEdit(!chatEdit);
+    }
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    handleSubmitButtonClick();
+  };
+  const onChange = (e) => {
+    const nextForm = e.target.value;
+    setText(nextForm);
+  };
```

Comment:
```suggestion
  const onChange = (e) => {
    setText(e.target.value);
  };
```

그냥 이렇게 하셔도 될 것 같습니다!